Xamarin.Forms.Platform.iOS/Renderers/PickerRenderer.cs

```diff
@@ -280,6 +281,11 @@ void UpdateVerticalTextAlignment()
 			Control.VerticalAlignment = Element.VerticalTextAlignment.ToNativeTextAlignment();			
 		}
 
+		void UpdateFlowDirection()
+		{
+			(Control as UITextField).UpdateTextAlignment(Element);
```

Comment:
Similar thing here, could you use Control.UpdateFlowDirection(Element) instead?

Reply:
Done.